### DISCUSSION TITLE
Custom dataset with streaming=True

### DISCUSSION THREAD

**Discussion by lakshmi-speak** [huggingface/datasets#4616] [Q&A]
Using a custom download method for my dataset, and setting `streaming=True` I get: 
```
AttributeError: 'StreamingDownloadManager' object has no attribute 'download_custom'
```
Is there a workaround for this?

**Comment by mariosasko** [comment]
Hi! `download_custom` is not available in the streaming mode indeed. We should raise `NotImplementedError` to make this clear. Can you provide more info on what you are trying to achieve with `download_custom` in the streaming mode so we can give you some workarounds?
